spec/fpm/package/python_spec.rb

```diff
@@ -164,24 +199,17 @@ def python_usable?
 
   context "python_scripts_executable is set" do
     it "should have scripts with a custom hashbang line" do
-      #subject.attributes[:python_install_bin] = '/usr/bin'
       subject.attributes[:python_scripts_executable] = "fancypants"
       subject.input("django")
 
-      # Get the default scripts install directory and use it to find django-admin.py from Django
-      # Then let's make sure the scripts executable setting worked!
-      if /darwin/ =~ RUBY_PLATFORM
-        # OSX
-        django_bindir = %x{python -c 'from distutils.sysconfig import get_config_var; print(get_config_var("BINDIR"))'}.chomp
-      elsif /cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM
-        # Windows
-        # (ptomulik) I'm not sure about this
-        django_bindir = %x{python -c 'from distutils.sysconfig import get_config_var; print(get_config_var("BINDIR"))'}.chomp
-      else
-        # Unix
-        #django_bindir = '/usr/local/bin'
-        django_bindir = %x{python -c 'from distutils.sysconfig import get_config_var; print(get_config_var("BINDIR"))'}.chomp
+      django_bindir = nil
+      # Determine, where 'easy_install' is going to install scripts
+      execmd('python', code_easy_install_default_script_dir) do |code,out,err|
```

Comment:
I think, it may be good to leave ``execmd`` as, in opposite to ``safesystem*`` ones, it provides all standard communication channels to the executed command (``stdin``, ``stdout``, ``stderr``, ``exit_status``). As you may se, I managed to implement both versions of ``safesystem*`` on top of ``execmd``. I'm ok with moving the python code to fixture file (this could push me to organize it little bit better), but the actuall script consists of two pieces -- [definition of a class](https://github.com/ptomulik/fpm/blob/fix/python_spec/spec/fpm/package/python_spec.rb#L15) and [a code that uses this class](https://github.com/ptomulik/fpm/blob/fix/python_spec/spec/fpm/package/python_spec.rb#L45). I think, the usage code (second part) is not worth giving it a fixture (it's a very small piece of code, and I don't think it's reusable), but hopefully it can be just passed to python interpreter via CLI.